docs/en/ingest-management/elastic-agent/configuring-kubernetes-metadata.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[configuring-kubernetes-metadata]]
+= Configuring Kubernetes metadata enrichment on {agent}
+
+Kubernetes https://www.elastic.co/guide/en/observability/current/monitor-kubernetes.html#beats-metadata[metadata] refer to contextual information extracted from Kubernetes resources. Metadata information enrich metrics and logs
+collected from a Kubernetes cluster, enabling deeper insights into Kubernetes environments.
+
+When the {agent}'s policy includes https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] which configures the collection of Kubernetes related metrics and container logs, the mechanisms used for the metadata enrichment are:
+
+* <<kubernetes-provider>> for log collection
+* Kubernetes metadata enrichers for metrics
+
+In case the {agent}'s policy does not include Kubernetes integration, but {agent} runs inside a Kubernetes
+environment the Kubernetes metadata are collected by the <<add_kubernetes_metadata-processor>>. The processor is configurable when {agent} is managed by {fleet}.
```

Comment:
```suggestion
In case the {agent}'s policy does not include the Kubernetes integration, but {agent} runs inside a Kubernetes
environment, the Kubernetes metadata are collected by the <<add_kubernetes_metadata-processor>>. The processor is configurable when {agent} is managed by {fleet}.
```
Same as above, plus I'd add a comma here.